src/service/data-util.service.ts

```diff
@@ -48,7 +48,10 @@ export class JhiDataUtils {
      * Method to open file
      */
     openFile(type: string, data: string) {
-        window.open('data:' + type + ';base64,' + data, '_blank', 'height=300,width=400');
+        const fileURL = 'data:' + contentType + ';base64,' + field;
```

Comment:
@sdoxsee something wrong here, where does `contentType` and `field` come from?

Reply:
This fails build, but travis didnt run ngc so i'm adding that step

Reply:
Sorry @deepu105 , you're totally right. I'd copy and pasted from my temporary fix to my own jhipster project since I couldn't easily change the ng-jhipster dependency itself (note: i wish the server's jhipster library and client's ng-jhipster libraries didn't exist and all the source was in jhipster). That said this is my bad. Fixing it now

Reply:
dont worry i fixed it already